### PR TITLE
Updating golang version to 1.15

### PR DESCRIPTION
**Pull request by girishramnani**
**What type of PR is this?**

> Documentation changes: Please include [skip ci] in your commit message as well
> /kind documentation
> [skip ci]

**What does does this PR do / why we need it**:

**Which issue(s) this PR fixes**:

Currently the prow is failing, seems like golang 1.13 was deprecated.

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [ ] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: scripts/rpm-prepare.sh [hunk 1]
```diff
@@ -13,8 +13,8 @@ export ODO_RPM_VERSION=${ODO_VERSION//-}
 
 # Golang version variables, if you are bumping this, please contact redhat maintainers to ensure that internal
 # build systems can handle these versions
-export GOLANG_VERSION=${GOLANG_VERSION:-1.13}
```
Comment (kadel) [inline scripts/rpm-prepare.sh:16]: I'm afraid that this change won't be possible right now. Build root that we use on Brew doesn't include go 1.15 :-( cc: @mohammedzee1000

Comment (mohammedzee1000) [reply scripts/rpm-prepare.sh:16]: Lets go ahead

Comment (mohammedzee1000) [reply scripts/rpm-prepare.sh:16]: I have already requested upgrade in brew. Worst case I can override envs

**Review by mohammedzee1000 — APPROVED** [review]
looks good
/lgtm

**Pull request merged**